src/Uno.UI.Tests/Windows_UI_Xaml_Markup/XamlReaderTests/Given_XamlReader.cs

```diff
@@ -1096,6 +1097,91 @@ public void When_xBind_TwoWay_Back()
 			Assert.AreEqual(true, r.MyVM.MyBool);
 		}
 
+		[TestMethod]
+		public void When_Collection_Implicit_Add_Item()
+		{
+			var SUT = LoadXaml<SwipeItems>(@"
+				<SwipeItems>
+					<SwipeItem Text='asd' />
+				</SwipeItems>
+			");
+
+			Assert.AreEqual(1, SUT.Count);
+			Assert.AreEqual("asd", SUT[0].Text);
+		}
+
+		[TestMethod]
+		public void When_Collection_Property_Nest_Collection()
+		{
+			var SUT = LoadXaml<SwipeControl>(@"
+				<SwipeControl>
+					<SwipeControl.LeftItems>
+						<SwipeItems Mode='Execute'>
+							<SwipeItem Text='asd' />
+						</SwipeItems>
+					</SwipeControl.LeftItems>
+				</SwipeControl>
+			");
```

Comment:
Why did add a inline loading instead of separate files?

Reply:
They are mostly really short snippets. I thought it would be better this way: no need to jump between files, or name them.

Reply:
If you above, there are a couple of inline xaml strings in this class already.

Reply:
Those are for error validation, mainly... anyways, not a big deal. It's still best if the whole file stays in the same model in the future.

Reply:
I agree with @jeromelaban. But we need it for Figma :-(